test/built-ins/RegExp/S15.10.2_A1_T1.js

```diff
@@ -20,20 +20,20 @@ var TextSE = "[^<]+";
 var UntilHyphen = "[^-]*-";
 var Until2Hyphens = UntilHyphen + "([^-]" + UntilHyphen + ")*-";
 var CommentCE = Until2Hyphens + ">?";
-var UntilRSBs = "[^]]*]([^]]+])*]+";
-var CDATA_CE = UntilRSBs + "([^]>]" + UntilRSBs + ")*>";
+var UntilRSBs = "[^]\\]*\\]([^]\\]+\\])*\\]+";
```

Comment:
I think this should be `[^\\]]*\\](?:[^\\]]+\\])*\\]+` instead. This seems to have been taken from [http://www.cs.sfu.ca/~cameron/REX.html](Robert D. Cameron's website) where there are also non-JavaScript versions. Those make the intention clearer.

Similar thing applies to `CDATA_CE` and `MarkupDeclCE`, too.